net/Socket.cpp

```diff
@@ -51,16 +51,15 @@
 #include <openssl/x509v3.h>
 #endif
 #include "WebSocketHandler.hpp"
-#include <net/HttpRequest.hpp>
-#include <NetUtil.hpp>
```

Comment:
I wonder why change the include style as part of this?

Reply:
yes .. and I remember that the system include style was not recommended for project includes. not thoroughly applied.

Reply:
```
-#include <net/HttpRequest.hpp>
-#include <NetUtil.hpp>
+#include "net/HttpRequest.hpp"
+#include "net/NetUtil.hpp"
```